packages/react-core/src/components/Badge/Badge.tsx

```diff
@@ -15,12 +17,14 @@ export const Badge: React.FunctionComponent<BadgeProps> = ({
   isRead = false,
   className = '',
   children = '',
+  screenReaderText,
   ...props
 }: BadgeProps) => (
   <span
     {...props}
     className={css(styles.badge, (isRead ? styles.modifiers.read : styles.modifiers.unread) as any, className)}
   >
+    {screenReaderText && <span className="pf-u-screen-reader">{screenReaderText}</span>}
```

Comment:
```suggestion
    {screenReaderText && <span className="pf-screen-reader">{screenReaderText}</span>}
```

I'm also wondering if it would make more sense to place this before or after the normal badge content... Using a badge to display unread messages for example, "7 unread message" sounds slightly better than "Unread messages 7", though that may not be the case for every use of a badge depending on the context.

@nicolethoen what do you think about the placement?

Reply:
Updated the classname, but I'm not marking this as resolved.

Reply:
As long as the screen reader text is 100% overridable, then whatever sounds most natural to an english speaker is likely what we should choose.